packages/core/lib/patchers/http_p.js

```diff
@@ -138,7 +139,11 @@ function enableCapture(module, downstreamXRayEnabled, subsegmentCallback) {
         subsegment.close(e);
       }
 
-      if (this._events && this._events.error && this._events.error.length === 1) {
+      // Only need to remove our listener & re-emit if we're not listening using the errorMonitor,
+      // e.g. the app is running on Node 10. Otherwise the errorMonitor will re-emit automatically.
+      // See: https://github.com/aws/aws-xray-sdk-node/issues/318
+      // TODO: Remove this logic once node 12 support is deprecated
```

Comment:
Do you mean "once 10 support is deprecated"?

Reply:
Do you mind explaining more on why do we remove the error listener  when `listenerCount('error')` returns 0?